Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -274,8 +274,13 @@ private final class TokenStreamCreator: SyntaxVisitor {
   }
 
   override func visit(_ node: IfConfigClauseSyntax) {
-    after(node.poundKeyword, tokens: .break)
-    after(node.condition?.lastToken, tokens: .newline)
+    switch node.poundKeyword.tokenKind {
+    case .poundIfKeyword, .poundElseifKeyword:
+      after(node.poundKeyword, tokens: .break)
+    default: ()
```

Comment:
Nit: `default: break` is clearer/more common than `default:` followed by a void expression.

Reply:
In fact, the only tokens that should occur here are `#if`, `#elseif`, and `#else`, right (since `#endif` is handled elsewhere)? Should we explicitly write `case .poundElseKeyword: break` and then have some logging in the default case to say that we saw something unexpected?

Reply:
I made the `default: break` change, and to explicitly `break` for `#else`.

As far as I know, only `#if`, `#elseif`, and `#else` should appear in `IfConfigClauseSyntax`. I'm not sure what to do by way of logging. Maybe throw an exception? Also, I don't imagine that anything other than the correct keywords would end up in `node.poundIfKeyword`. Other pound-keywords would probably be parsed differently.

Reply:
For now, maybe use `preconditionFailure`? That's not a great long-term solution because if a new token was introduced we'd want to handle it better than crashing, but we should probably do that for now and then think about a better long-term solution for how we log unexpected things gracefully during formatting.